libs/wingsdk/test/target-sim/api.test.ts

```diff
@@ -25,6 +26,8 @@ const INFLIGHT_CODE_WITH_CONTENTTYPE_RESPONSE_HEADER = `async handle(req) { retu
 // Handler that responds to a request without a response body
 const INFLIGHT_CODE_NO_BODY = `async handle(req) { return { status: 200 }; }`;
 
+const _platformManager = new PlatformManager({ platformPaths: ["sim"] });
```

Comment:
Can this be removed?

Reply:
Yupp, this needs to use the helper methods 😅 good call